File: Semestr3/AnNumeryczna/cwiczenia/lista10/zad5.jl
setprecision(1000)
Fl = BigFloat

X = [0; 1; 2; 4; 6]
F = [1; 9; 23; 93; 259]
X = map(Fl, X)
F = map(Fl, F)

A = Array{Fl}(4,4)
for i = 1:4, j = 1:4
    A[i,j] = sum([X[k]^(i+j-2) for k = 1:5])
end

B = Array{Fl}(4,5)
for i = 1:4, j = 1:5
    B[i,j] = X[j]^(i-1)
end
BF = B*F
for i in \(A,BF)
    @printf "%.3e\n" i
end

w(x) = x^3 + 7x + 1

print(w(6)) 

File: Semestr5/weppo/lista3/zad5.js
function sum(...args) {
    let s = 0
    for(let arg of args) {
        s += arg
    }
    return s
}
console.log(sum())
console.log(sum(1,2,3))
console.log(sum(1,2,3,4))
console.log(sum(1,2,3,4,5))